packages/mobile-app/wallet/db.ts

```diff
@@ -0,0 +1,70 @@
+import { Kysely, Generated, Migrator } from "kysely";
```

Comment:
suggestion: Super nit, but IMO it would make sense to put this db layer in a folder (e.g. /mobile-app/db/wallet). Otherwise we'll have a bunch of folders named after resources at the project root which could be confusing.

Reply:
Discussed this on Slack